docs/docs/schema/pipeline.json

```diff
@@ -424,15 +496,18 @@
         },
         "KafkaSinkConnector": {
             "title": "KafkaSinkConnector",
+            "description": "Kafka sink connector model",
             "type": "object",
             "properties": {
                 "type": {
                     "title": "Type",
+                    "description": "Component type,",
```

Comment:
is it possible to remove the trailing comma in the description?

Reply:
Must be fixed now, good catch